packages/ember-intl/addon-test-support/setup-intl.ts

```diff
@@ -23,15 +22,15 @@ export function setupIntl(
   hooks.beforeEach(async function (this: TestContext) {
     const { owner } = getContext() as TestContext;
 
-    const intl = owner.lookup('service:intl') as IntlService;
+    const intl = owner.lookup('service:intl');
 
     intl.setLocale(locale);
     intl.setOnMissingTranslation(onMissingTranslation);
 
     if (translations) {
-      addTranslations(locale, translations);
+      await addTranslations(locale, translations);
+    } else {
+      await settled();
```

Comment:
Note, [`addTranslations()` calls `await settled()`](https://github.com/ember-intl/ember-intl/blob/v7.1.3/packages/ember-intl/addon-test-support/add-translations.ts#L23).

To fix the lint error, I went with an if-else statement so that `setupIntl()` doesn't call `await settled()` twice in the if-case.